docs/examples/gnmi-clients/gnmic/index.md

```diff
@@ -114,9 +1056,11 @@ gnmic -a 127.0.0.1:6030 -u admin -p admin --insecure get \
   --path ".../config/..." >> outputs/configs.json
 ```
 
-## Smash paths
-
 ```shell
-gnmic -a 127.0.0.1:6030 -u admin -p admin --insecure get \
-  --path eos_native:/Smash >> outputs/smashes.json
+gnmic -a 127.0.0.1:6030 -u admin -p admin --insecure get --path \
+  /network-instances/network-instance >> outputs/network-instances.json
+
+gnmic -a 127.0.0.1:6030 -u admin -p admin --insecure get --path \
```

Comment:
these should probably be split into 2 backtick regions so that folks can copy them independently.